components/coding/studentPortal/AssignmentInputBox.tsx

```diff
@@ -49,7 +50,13 @@ export const AssignmentInputBox: React.FC<AssignmentInputBoxProps> = ({
               variables: {
                 objects: submissionVariables,
               },
+              onCompleted: () => {
```

Comment:
Not incorrect but would be cleaner if you move this onComplete above where you call useMutation to define the saveAssignmentInput function. You can also move the variables object up there

Reply:
Done